microservices/matchmaking-service/src/scrim/scrim.service.ts

```diff
@@ -191,12 +191,22 @@ export class ScrimService {
 
         await this.scrimCrudService.removeScrim(scrimId);
         scrim.status = ScrimStatus.CANCELLED;
-        await this.eventsService.publish(EventTopic.ScrimCancelled, scrim, scrim.id);
+        await this.eventsService.publish(
+            EventTopic.ScrimCancelled,
+            {
+                ...scrim,
+                cancelReason: reason,
+            },
+            scrim.id,
+        );
 
         this.analyticsService
             .send(AnalyticsEndpoint.Analytics, {
                 name: "scrimCancelled",
-                strings: [["scrimId", scrim.id]],
+                strings: [
+                    ["scrimId", scrim.id],
+                    ["reason", reason ?? "none provided"],
```

Comment:
Rather than `none provided`, just leave it undefined. The UI/discord embed can just leave it off or show a default string there.

Reply:
Wasn't there an issue with analytics at one point? Couldn't remember if it was an empty string or undefined or something.

Reply:
Yeah, it doesn't like undefined.